libraries/plugins/witness/witness_plugin.cpp

```diff
@@ -53,15 +55,41 @@ using chain::account_object;
 
 void new_chain_banner(const scorum::chain::database& db)
 {
-    std::cerr << "\n"
-                 "********************************\n"
-                 "*                              *\n"
-                 "*   ------- NEW CHAIN ------   *\n"
-                 "*   -   Welcome to Scorum!  -   *\n"
-                 "*   ------------------------   *\n"
-                 "*                              *\n"
-                 "********************************\n"
-                 "\n";
+    const std::string welcome = "Welcome to Scorum!";
+    const std::string info_ch = "NEW CHAIN";
+    const char space_symbol = ' ';
+    const char border_symbol = '*';
+
+    size_t sz = std::max(welcome.size(), info_ch.size());
+    sz *= 3;
+    cli::formatter p(sz);
+    p.print_line(border_symbol);
```

Comment:
this implementation looks heavy, over complicated for me.

CliBuffer().println("NEW CHAIN")
                .println("Welcom to Scorum")
                .frame("*")
                .margin(1);